include/network.hpp

```diff
@@ -11,33 +11,33 @@
 
 class Network {
  protected:
-  int nlayers_global; /* Total number of Layers of the network */
-  int nlayers_local;  /* Number of Layers in this network block */
+  int nlayers_global; // Total number of Layers of the network 
+  int nlayers_local;  // Number of Layers in this network block 
 
-  int nchannels;   /* Width of the network */
-  MyReal dt;       /* Time step size */
-  MyReal loss;     /* Value of the loss function */
+  int nchannels;   // Width of the network 
+  MyReal dt;       // Time step size 
+  MyReal loss;     // Value of the loss function 
   MyReal accuracy; /* Accuracy of the network prediction (percentage of
                       successfully predicted classes) */
 
-  int startlayerID; /* ID of the first layer on that processor */
-  int endlayerID;   /* ID of the last layer on that processor */
+  int startlayerID; // ID of the first layer on that processor 
+  int endlayerID;   // ID of the last layer on that processor 
 
-  int ndesign_global;   /* Global number of design vars  */
-  int ndesign_local;    /* Number of design vars of this local network block  */
+  int ndesign_global;   // Global number of design vars  
+  int ndesign_local;    // Number of design vars of this local network block  
   int ndesign_layermax; /* Max. number of design variables of all hidden layers
```

Comment:
In my opinion, these comments should be either all /* */ or all //, no matter if single or multiple line. Don't like the switch.